donatos_andrea/index.js

```diff
@@ -1,11 +1,82 @@
+const memberName = document.querySelector("#member_name");
+const commentArea = document.querySelector("#comment_area");
+const commentButton = document.querySelector("#comment_button");
+
+const commentList = document.querySelector("#comment_list");
+const commentListData = [
+	{
+		name: "Jheferson",
+		comment: `Amazing goal, keep it up!`,
+		date: new Date("Sat Apr 27 22:28:24 2024")
+	},
+	{
+		name: "Von",
+		comment: `Your strategic approach to planning and
+        executing your life goals is impressive`,
+		date: new Date("Sat Apr 27 22:45:41 2024")
+	},
+	{
+		name: "Maui",
+		comment: `I love your goals, they reflect mine`,
+		date: new Date("Sat Apr 27 23:02:34 2024")
+	},
+	{
+		name: "Mark",
+		comment: `You could survive a zombie apocalypse`,
+		date: new Date("Sat Apr 27 23:06:32 2024")
+	}
+];
+
 function commentForm() {
-    let comment_button = document.getElementById("comment_button");
-    let name = document.getElementById("member_name");
-    let comment = document.getElementById("comment_area");
-
-    if (name.value.length > 0 && comment.value.length > 0) {
-        comment_button.disabled = false;
-    } else {
-        comment_button.disabled = true;
-    }
-} 
+	 if (memberName.value && commentArea.value) {
+         commentButton.disabled = false;
+     } else {
+         commentButton.disabled = true;
+}
```

Comment:
Hi! Please add indentation to this closing brace in the code so that it aligns properly.